.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+ - 6.10.2
+sudo: required
+script:
+ - npm index.js
```

Comment:
What will this script do?

It would be better to setup style guide in the project and run style checks for now.